rest/src/main/java/discord4j/rest/request/GlobalRateLimiter.java

```diff
@@ -43,7 +43,7 @@
 
     private static final Logger log = Loggers.getLogger(GlobalRateLimiter.class);
 
-    private final Semaphore outer = new Semaphore(8, true);
+    private final Semaphore outer = new Semaphore(14, true);
```

Comment:
Can you expose this value through a constructor parameter named `parallelism` ?

I will add code to further expose the value after we merge it (through RouterOptions). This could be a good value for your use case, but I'm not sure if we're changing the current default considering this one hits the global limit.

I confirmed 14 optimizes the throughput but I have no idea why 😛